### PR TITLE
JPERF-811: Show how flaky shouldTolerateEarlyFinish is, so that it's …

### PR DESCRIPTION
**Pull request by pczuj**
…clear that next change fixes it

### CODE REVIEW

**Comment by pczuj** [comment]
The test will be fixed in #23

File: src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshTest.kt [hunk 1]
```diff
@@ -56,10 +56,12 @@ class SshTest {
         SshContainer().useSsh { sshHost ->
             installPing(sshHost)
 
-            val fail = sshHost.runInBackground("nonexistent-command")
-            val failResult = fail.stop(Duration.ofMillis(20))
+            repeat(1000) {
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshTest.kt:59]: Good change, exposes the problem more predictably :+1:

**Comment by dagguh** [comment]
This PR left an unmaintained branch. It's best to open PRs from forks.

**Comment by pczuj** [comment]
> This PR left an unmaintained branch.

No, I believe I removed the branch. Don't see it in the picker:
Screenshot 2022-06-29 at 14 39 48

> It's best to open PRs from forks.

I don't see any advantage and it's easier to access other's changes with vanilla git client when most of the PR branches land in the main repository. I'm open to discussion about it. Maybe I don't know about something.

**Comment by dagguh** [comment]
Ah right, it's a different branch :+1:

**Pull request closed without merging**